plasmapy/physics/tests/test_parameters.py

```diff
@@ -137,6 +137,22 @@ def test_Alfven_speed():
     with pytest.raises(UserWarning):
         assert Alfven_speed(1.0, n_i) == Alfven_speed(1.0 * u.T, n_i)
 
+    # tests for z_mean functionality
+    # testing if warning is signaled for default value
+    with pytest.warns(PhysicsWarning):
+        Alfven_speed(1 * u.T, 5e19 * u.m**-3, ion='p')
+    # testing for user input z_mean
+    testMeth1 = Alfven_speed(1 * u.T,
+                             5e19 * u.m**-3,
+                             ion='p',
+                             z_mean=0.8).si.value
```

Comment:
The name may not be the most fortunate here, as testing meth could probably be delegated to PyChemistry. Or Pysenberg, or something. ;)

The line itself is completely fine though and I'm fine with keeping this as it is!

Reply:
Well I am in Albuquerque so Pysenberg would be quite appropriate 😷 💰 💵 🔬